### ISSUE TITLE
Colour contrast problem for Carbon tag component

### ISSUE THREAD

**Issue by KJaspers** [carbon-design-system/carbon#6997]
Our colour contrast analyser found the following problem for the Carbon tag component (kind=purple):

![Screenshot 2020-10-08 at 08 33 37](https://user-images.githubusercontent.com/50624181/95423285-1ed91700-0941-11eb-96c3-fe3b81a62826.png)

**Comment by tw15egan** [comment]
cc @carbon-design-system/design

**Comment by laurenmrice** [comment]
For the purple tag:
-The text color is `Purple 70` (#6929C4)
-The background color is `Purple 20` (#E8DAFF)

This combination does pass WCAG AA. I think you may have the wrong hex value for the text color (#695CD5)

**Comment by laurenmrice** [comment]
Its a bug in react which we need to update to the above ^

**Comment by tw15egan** [comment]
@laurenmrice that's what we currently have in React

<img width="299" alt="Screen Shot 2020-10-08 at 12 22 02 PM" src="https://user-images.githubusercontent.com/11928039/95503906-e0ebeb00-0960-11eb-89ed-7938d4974b98.png">

**Comment by laurenmrice** [comment]
Oh, you are right ! Doing some further investigation, I looked at this in Firefox/Chrome/Safari, the Firefox browser is making the colors show up differently (almost a warmer color).
![Artboard](https://user-images.githubusercontent.com/43969356/95505680-c162bb80-097c-11eb-9174-ee94822b0fe7.png)
.

**Comment by KJaspers** [comment]
BTW: We are using the Carbon Vue.js library.

**Comment by joshblack** [comment]
Is there a good way to reproduce the issue? Tried using FF's built-in contrast checker as well and saw what seems to be the correct values:

![Screen Shot 2020-10-19 at 11 38 55](https://user-images.githubusercontent.com/3901764/96480488-e9b1ac00-11ff-11eb-9924-e68a1045b3a8.png)

**Comment by tw15egan** [comment]
Closing this out, contrast values were increased in this PR: https://github.com/carbon-design-system/carbon/pull/7262
